src/views/download/download.jsx

```diff
@@ -0,0 +1,180 @@
+                                            {' '}<a href="https://scratch.mit.edu/scratchr2/static/sa/Scratch2StarterProjects.zip">
+                                                <FormattedMessage id='download.download' />
+                                            </a>
+                                        </li>
+                                        <li className="installation-downloads-item">
+                                            <FormattedMessage id='download.gettingStarted' /> -
+                                            {' '}<a href="https://cdn.scratch.mit.edu/scratchr2/static/__709da8e5f3d72129538a4ccdbcbf5f2a__/pdfs/help/Getting-Started-Guide-Scratch2.pdf">
+                                                <FormattedMessage id='download.download' />
+                                            </a>
+                                        </li>
+                                        <li className="installation-downloads-item">
+                                            <FormattedMessage id='download.scratchCards' /> -
+                                            {' '}<a href="https://cdn.scratch.mit.edu/scratchr2/static/__709da8e5f3d72129538a4ccdbcbf5f2a__/pdfs/help/Scratch2Cards.pdf">
+                                                <FormattedMessage id='download.download' />
+                                            </a>
+                                        </li>
+                                    </ul>
+                                </div>
+                            </FlexRow>
+                        </section>
+
+                        <section id="updates">
+                            <span className="nav-spacer"></span>
+                            <h2><FormattedMessage id='download.updatesTitle' /></h2>
+                            <p><FormattedMessage id='download.updatesBody' /></p>
+                            <p><FormattedMessage id='download.currentVersion' /></p>
+                        </section>
+
+                        <section id="other">
+                            <span className="nav-spacer"></span>
+                            <h2><FormattedMessage id='download.otherVersionsTitle' /></h2>
+                            <p><FormattedHTMLMessage id='download.otherVersionsOlder' /></p>
+                            <p><FormattedHTMLMessage id='download.otherVersionsAdmin' /></p>
+                        </section>
+
+                        <section id="issues">
+                            <span className="nav-spacer"></span>
+                            <h2><FormattedMessage id='download.knownIssuesTitle' /></h2>
+                            <p><FormattedMessage id='download.knownIssuesOne' /></p>
+                            <p><FormattedMessage id='download.knownIssuesTwo' /></p>
+                            <p><FormattedHTMLMessage id='download.knownIssuesThree' /></p>
+                            <a href="https://scratch.mit.edu/discuss/3/" className='button'>
```

Comment:
this is currently showing blue text on a blue button – you may need to change the className here to `className='button mod-link` and change the text color, as it appears the text color from the `a` tag is overriding the class's color